lib/collections/addon/components/collection-item-picker/component.ts

```diff
@@ -89,15 +89,20 @@ export default class CollectionItemPicker extends Component {
 
         // Filter out nodes that are already in the current collection
         const nodeIds = nodes.mapBy('id').join();
-        const unsubmittableReviewsStates = [
-            CollectionSubmissionReviewStates.Accepted, CollectionSubmissionReviewStates.Pending,
-        ].join();
-        const cgm = await this.collection.queryHasMany('collectionSubmissions', {
+        // making two calls since the API doesn't    OR filtering yet
+        const cgmAccepted = await this.collection.queryHasMany('collectionSubmissions', {
```

Comment:
What happens if there are more than 10 nodes returned by this?

Reply:
The user will be able to select that project and possibly try to submit it. Let me see if there is a reasonable error message shown if that happens.